polaris-react/src/components/Scrollable/Scrollable.tsx

```diff
@@ -144,6 +154,9 @@ const ScrollableComponent = forwardRef<ScrollableRef, ScrollableProps>(
       horizontal && styles.horizontal,
       shadow && topShadow && styles.hasTopShadow,
       shadow && bottomShadow && styles.hasBottomShadow,
+      scrollbarWidth && styles[variationName('scrollbarWidth', scrollbarWidth)],
+      scrollbarGutter &&
+        styles[toCamelCase(variationName('scrollbarGutter', scrollbarGutter))],
```

Comment:
This doesn't need to change, but it took me a minute to understand why the `toCamelCase` utility was needed here and to figure out the `variationName` function.

Instead of relying on the value of the `scrollbarGutter` to match to the corresponding class, could we use a simple object map here maybe? I haven't tested this, but I would thing the typings might better 🤷 

```ts
const scrollbarGutterClasses: { [k: ScrollableProps['scrollbarGutter']: string } = {
  stable: style.scrollbarGutterStable,
  'both-edges': style.scrollbarGutterBothEdges,
  auto: style.scrollbarGutterAuto
}
```

I used `both-edges`, but that was a personal preference. `stable both-edges` is more accurate, but the shortened version felt like a better DX since `stable` is [always required](https://developer.mozilla.org/en-US/docs/Web/CSS/scrollbar-gutter#formal_syntax) with `both-edges`